playbooks/adjust-cinder-backend.yml

```diff
@@ -111,6 +117,11 @@
 
   - name: trigger stop all cinder services
     meta: flush_handlers
+    
+
+  - name: wait for cinder services down
+    shell: sleep 10s
+    when: set_cinder_host or update_cinder_backend
```

Comment:
No, I meant using the `command` module and directly executing a sensu plugin to check the status of the cinder service. Like https://github.com/blueboxgroup/ursula/blob/master/roles/percona-server/tasks/main.yml#L67-L72